content/news/2022-10-09-ubucon2022-sponsorship/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "UbuCon Asia 2022 Sponsorship"
+date: 2022-10-09T12:51:02+08:00
+draft: false
+authors:
+    - name: Robbi Nespu
+      bio: Sponsorship and Finances + Marketing (Global team) Ubucon Asia 2022
+      email: robbinespu@gmail.com
+      #launchpad: robbinespu
+      #github: robbinespu
+      profile: https://robbinespu.gitlab.io/images/upload/Screenshot_20210816_124946.png
+      linkurl: https://robbi.my
+      linklabel: 🌎 Personal Website
+---
+
+This year, we bring you UbuCon Asia 2022 as hybrid event. It will be physically held at **Nuritkum Square, Seoul, South Korea** and remotely from via Youtube (talks) and jitsi(BoF). Our target audience is obviously for people around Asia, but it is open for anyone around the globe to come and join us.
+
+Let's have a quick look at our first [UbuCon2021 event final report](https://2021.ubucon.asia/news/2021-12-01-ubucon-asia-2021-final-report/) as it shows that **25% of participants were students** and **60% were professionals** who work in the field. By looking at this number and statistic, truely this <mark>great opportunity for our sponsorship to find talents among the participants while also empowering your brand</mark> across the world.
+
+This year it is going to be bigger, exciting and more interactive than before! Because we are now a physical (and hybrid) event, it provides excellent opportunities for sponsors to come and open booths onsite, allowing you as sponsors to interact with participants in-person. I believe this is a great place to promote your company or services.
+
+UbuCon is a non-profit events by combining various open source communities across Asia and the event itself are focuses on Asia region that bring together many of Ubuntu users, developers, contributors to share their stories, experiences, discuss on Ubuntu, a popular modern linux distribution and its related free and open source technologies on server, desktop, cloud, IoT and much more. It is organized solely by group of volunteers without any financial compensation. Thus, UbuCon Asia depends on sponsors and donations. Sponsoring this event isn't just a way to help us achieve our goals but also a <mark>good way to show your sincere support on open source movement and Ubuntu community</mark> itself.
+
+We have created various packages for sponsorship, as you can find on our [Sponsorship Prospectus](https://2022.ubucon.asia/become-a-sponsor/uca22_sponsorship_prospectus_en.pdf). Don’t hesitate to contact sponsorship team ([sponsorship@ubucon.asia](mailto:sponsorship@ubucon.asia)) to discuss additional sponsorship opportunities. We can work together to find the right package for your organization.
```

Comment:
```suggestion
We have created various packages for sponsorship, as you can find on our [Sponsorship Prospectus](../../become-a-sponsor/uca22_sponsorship_prospectus_en.pdf). Don’t hesitate to contact sponsorship team ([sponsorship@ubucon.asia](mailto:sponsorship@ubucon.asia)) to discuss additional sponsorship opportunities. We can work together to find the right package for your organization.
```

Considering the localized page path, relative path would be better

Reply:
ok, changed